app-template/copay/appConfig.json

```diff
@@ -36,6 +36,6 @@
     "debitcard": false,
     "shapeshift": false,
     "giftcards": false,
-    "walletConnect": false
+    "walletConnect": true
```

Comment:
Are you going to support it on Copay app too?

Reply:
yess